sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala

```diff
@@ -105,7 +105,7 @@ class RocksDBStateStoreSuite extends StateStoreSuiteBase[RocksDBStateStoreProvid
 
       // Create state store in a task and get the RocksDBConf from the instantiated RocksDB instance
       val rocksDBConfInTask: RocksDBConf = testRDD.mapPartitionsWithStateStore[RocksDBConf](
-        spark.sqlContext, testStateInfo, testSchema, testSchema,
+        spark, testStateInfo, testSchema, testSchema,
```

Comment:
Perhaps a follow-up should be to weed out all existing uses of SQLContext in the code base. There are a couple of user facing APIs that we can't change, but most of it should go.

Reply:
For now I reverted this change. Let's to it in another PR.